### ISSUE TITLE
null/bad magic number output

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#7]
```
What steps will reproduce the problem?
1. use setUrls(ClasspathHelper.getUrlsForCurrentClasspath());
2. create a new Reflections object.

What is the expected output? What do you see instead?
I would expect no output at all. lots of odd output on standard output that
looks like error output.

What version of the product are you using? On what operating system?
0.9.1, Linux 2.6.26-1-amd64 #1 SMP Wed Nov 26 18:26:02 UTC 2008 x86_64
GNU/Linux 

Please provide any additional information below.
I used the following dependency libraries: google-collect-snapshot-20080820
and javassist-3.9.0.

I checked out the source code and found the source of output:

In the JavasistAdapter

public Iterator<ClassFile> iterateClasses(final Collection<URL> urls, final
Filter<String> filter)
...
protected ClassFile computeNext()
...
catch (IOException e) {
                            System.out.println(e.getMessage());
//                            throw new RuntimeException(e);
                        }

I looked into this issue a little more and found why this happens. The
problem is that the virtualFileIterator returns all files (<filename>.*)
and directories(!) on the classpath, only class files should be handled here. 

I've added a very lame fix in the attached JavassistAdapter in attachment,
most likely there is a better way.



```

Original issue reported on code.google.com by `teyckmans@gmail.com` on 19 Dec 2008 at 4:26

Attachments:
- [unexpected-stdout](https://storage.googleapis.com/google-code-attachments/reflections/issue-7/comment-0/unexpected-stdout)
- [JavassistAdapter.java](https://storage.googleapis.com/google-code-attachments/reflections/issue-7/comment-0/JavassistAdapter.java)

**Comment by GoogleCodeExporter** [comment]
```
After going through the issues again, I think this is the same issue as issue 
6. 

Sorry for the double logging.
```

Original comment by `teyckmans@gmail.com` on 19 Dec 2008 at 4:41

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.9.2
```

Original comment by `ronm...@gmail.com` on 27 Dec 2008 at 3:13
- Changed state: **Fixed**
